src/EnergyPlus/IOFiles.cc

```diff
@@ -442,6 +444,25 @@ void IOFiles::OutputControl::getInput(EnergyPlusData &state)
             }
         }
     }
+
+    auto const timestamp_instances = ip->epJSON.find("OutputControl:Timestamp");
+    if (timestamp_instances != ip->epJSON.end()) {
+        auto const &instancesValue = timestamp_instances.value();
+        for (auto instance = instancesValue.begin(); instance != instancesValue.end(); ++instance) {
```

Comment:
This is a unique object, so this 'loop' is fine.  It would be kinda nice if unique objects weren't stored as an array, but that would probably make things harder in general.  So here we are, and a loop over zero or one objects is fine.